platform_plugin_elm_credentials/api/views.py

```diff
@@ -91,12 +96,27 @@ def get(self, request, course_id: str) -> HttpResponse:
                 status_code=status.HTTP_404_NOT_FOUND,
             )
 
-        credential_builder = CredentialBuilder(course, user, certificate)
-        data = ELMv3DataModel(**credential_builder.build())
+        try:
+            query_params = QueryParamsModel(**request.query_params.dict())
+            additional_params = query_params.model_dump()
```

Comment:
can we make the params optional? So we have defaults ie. downloading the file and using a month as defaults.

Reply:
Yes, I modified the class that validates the query parameters. By default, it has these values.

```python
class QueryParamsModel(BaseModel):
    """Pydantic model for query parameters."""

    expired_at: Optional[datetime] = None
    to_file: bool = True
```